### PR TITLE
NavBar Button highlighted on relevant page #13

### LINKED ISSUE #13: NavBar Button does not change colour on corresponding page
Screenshot 2023-10-13 at 12 08 14 pm

### PR DESCRIPTION
**Pull request by Maiha192**
Closes #13 

Screenshot 2023-10-19 at 3 57 45 pm

### CODE REVIEW

File: src/components/NavBar.tsx [hunk 1]
```diff
@@ -21,6 +22,31 @@ function classNames(...classes: string[]) {
...
+  const [navItems, setNavItems] = useState(navigation);
+
+  useEffect(() => {
+    const path = window.location.pathname;
+    const updatedNavItems = navItems.map(item => {
+      if (item.href === path) {
+        return { ...item, current: true };
+      } else {
+        return { ...item, current: false };
+      }
```
Comment (bitcoinbrisbane) [inline src/components/NavBar.tsx]: Could we do this in 1 line, something like
```
return { ...item, current: item.href === path }
```

File: src/components/NavBar.tsx [hunk 2]
```diff
@@ -21,6 +22,31 @@ function classNames(...classes: string[]) {
...
+    setNavItems(updatedNavItems);
+  }, []);
+
+  const handleNavButtonClick = (name: string) => {
+    const updatedNavigation = navItems.map(item => {
+      if (item.name === name) {
+        return { ...item, current: true };
+      } else {
+        return { ...item, current: false };
+      }
```
Comment (bitcoinbrisbane) [inline src/components/NavBar.tsx]: and here

**Review by bitcoinbrisbane — CHANGES_REQUESTED** [review]
See comments

File: src/components/NavBar.tsx (hunk 1 above)
Comment (Maiha192) [reply src/components/NavBar.tsx]: Yes, we can. I have fixed this in new commit.

File: src/components/NavBar.tsx (hunk 2 above)
Comment (Maiha192) [reply src/components/NavBar.tsx]: I have fixed this in new commit too.

**Review by bitcoinbrisbane — APPROVED** [review]

**Pull request merged**